### ISSUE TITLE
Salt Formula password fields should not be logged

### ISSUE THREAD

**Issue by smox** [uyuni-project/uyuni#6725]
_Short description_

When I write a salt formula and use a password field, the input is logged to /var/log/rhn/rhn_web_api.log

```
servername:~ # grep -iHnR 'asdsadsadadasd' /var/log
/var/log/rhn/rhn_web_api.log:10337:[2023-03-13 11:15:57,367] INFO  - REQUESTED FROM: XX.XX.XXX.XX CALL: formulas.save(formula_name="active-directory", id="1000000000", type="SERVER", content={"ad":{"user":"Administrator","password":"asdsadsadadasd","realm":"ad.domain.com","dc":"dc.domain.com"}, ... } CALLER: (user1) TIME: 0.533 seconds
```

_Short description about the requested feature._

IMHO data from password fields should not be logged because most of the time it contains sensitive data.
I've seen that many programs replace the password with a "removed" string before logging it.
The problem is that I use this formula to join a server to an active directory domain and I do not want that my ad credentials are in the log files. I trust my co-worker but they also should not see my credentials.

The database inputs are not really a problem because after I join the server I can save the formula with dummy data but I do not want to search through all log files and remove my credentials.

**Comment by thesp0nge** [comment]
It's just a hint for the next time.

**Comment by witekest** [comment]
Fixed in Uyuni 2023.03
